README.md

```diff
@@ -32,4 +32,8 @@
     * popleft(): Removes a value from the front of the Deque and returns it. Raises an exception of the Deque is empty.
     * peek(): Returns the next value that would be returned by pop but leaves the value in the Deque. Returns None if the Deque is empty.
     * peekleft(): Returns the next value that would be returned by popleft but leaves the value in the Deque. Returns None if the Deque is empty.
-    * size(): Returns the count of items in the Deque. Returns 0 if the Deque is empty.
\ No newline at end of file
+    * size(): Returns the count of items in the Deque. Returns 0 if the Deque is empty.
+## Binary Tree:
+    * takes a bin_type: max heap (largest number is at the top of the tree) or min heap(smallest number is at the top of the tree).
+    * push(val): Puts a new value into the heap, while maintaining the heap property.
+    * pop(): Removes the "top" value in the heap, while maintaining the heap property.
```

Comment:
Let's adjust this to return the value stored there too.  Otherwise how would this structure be useful?